spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/deployer/ApplicationManifestDifference.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.domain.deployer;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.MapDifference;
+
+import org.springframework.util.Assert;
+
+/**
+ * Describes the difference between two ApplicationManifests
+ * @author Mark Pollack
+ */
+public class ApplicationManifestDifference {
```

Comment:
Why to move these from server to a shared package? Are we planning to use these in a client?

Reply:
I wanted to address that in another PR, but yes, I see them moving into the client package, but not in the domain package.